app/src/components/SearchBar.css

```diff
@@ -0,0 +1,8 @@
+input {
+  border: 0;
+  width: 100%;
+}
+
+input:focus {
```

Comment:
Hiding the outline of a focused search field is not good for accessibility
https://www.a11yproject.com/posts/never-remove-css-outlines/